recipes-bsp/u-boot/u-boot-common_2020.04.inc

```diff
@@ -12,4 +12,3 @@ SRCREV = "36fec02b1f90b92cf51ec531564f9284eae27ab4"
 SRC_URI = "git://github.com/u-boot/u-boot.git"
 
 S = "${WORKDIR}/git"
-B = "${WORKDIR}/build"
```

Comment:
Hi, Seems this is spurious change that breaks u-boot build, please resend without and I'll merge it

Reply:
I think that crept in when I was re-basing to dunfell.  It all got a bit mad for a minute there :¬)

Should be resolved now.